compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -1221,6 +1225,187 @@ public void visit(BLangForever foreverStatement) {
                 foreverStatement.addFunctionVariable(varSymbol);
             }
         }
+
+        List<Scope.ScopeEntry> localVariableList = new ArrayList<>(this.env.scope.entries.values());
+        for (Scope.ScopeEntry scopeEntry : localVariableList) {
+            if ("stream".equals(((scopeEntry).symbol.type.tsymbol).name.value)) {
+                foreverStatement.addFunctionVariable((BVarSymbol) scopeEntry.symbol);
+            }
+        }
+
+        //Validate output attribute names with stream/struct
+        for (StreamingQueryStatementNode streamingQueryStatement : foreverStatement.getStreamingQueryStatements()) {
+            checkOutputAttributes((BLangStatement) streamingQueryStatement);
+            validateOutputAttributeTypes((BLangStatement) streamingQueryStatement);
+        }
+    }
+
+    private void checkOutputAttributes(BLangStatement streamingQueryStatement) {
```

Comment:
Please move all your private method to the end of the class.

Reply:
I have moved all *my* private methods.